docs/DataPassthroughRequest.md

```diff
@@ -5,12 +5,13 @@
 
 Name | Type | Description | Notes
 ------------ | ------------- | ------------- | -------------
-**Method** | **string** |  | 
+**Method** | **MethodEnum** |  | 
 **Path** | **string** |  | 
 **BaseUrlOverride** | **string** |  | [optional] 
 **Data** | **string** |  | [optional] 
+**MultipartFormData** | [**List&lt;MultipartFormFieldRequest&gt;**](MultipartFormFieldRequest.md) | Pass an array of &#x60;MultipartFormField&#x60; objects in here instead of using the &#x60;data&#x60; param if &#x60;request_format&#x60; is set to &#x60;MULTIPART&#x60;. | [optional] 
 **Headers** | **Dictionary&lt;string, Object&gt;** |  | [optional] 
-**RequestFormat** | **string** |  | [optional] 
+**RequestFormat** | **RequestFormatEnum** |  | [optional] 
```

Comment:
I forgot, can c# handle enums?

Reply:
Oh good point, rip, this needs the enum + string treatment.